src/doc/book/structs.md

```diff
@@ -163,11 +163,48 @@ struct Point(i32, i32, i32);
 let black = Color(0, 0, 0);
 let origin = Point(0, 0, 0);
 ```
+
 Here, `black` and `origin` are not equal, even though they contain the same
 values.
```

Comment:
Maybe it should just say "are not the same type, even though they contain the same values".